Dockerfile-ubuntu-22.04

```diff
@@ -16,7 +16,10 @@ RUN apt-get install -y curl gnupg && \
 curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg && \
 mv bazel.gpg /etc/apt/trusted.gpg.d/ && \
 echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list && \
-apt-get update -y && apt-get install -y bazel
+apt-get update -y && apt-get install -y bazel-6.4.0
+
+# soft link bazel-6.4.0 to bazel
+RUN ln -s /usr/bin/bazel-6.4.0 /usr/bin/bazel
```

Comment:
is that package buggy? (and doesn't create the symlink by default)
if there is more than one version installed maybe using https://manpages.ubuntu.com/manpages/trusty/en/man8/update-alternatives.8.html would be cleaner?

Reply:
solved by using Bazelisk